store/elasticsearch/search.go

```diff
@@ -246,6 +274,7 @@ func (sr *Searcher) buildFilterQueries(query elastic.Query, filters map[string][
 			values = append(values, rawVal)
 		}
 
+		key := fmt.Sprintf("%s.keyword", key)
```

Comment:
this means field without `keyword` will not get matched?

Reply:
yes, this should be consistent with filtering in get all
https://github.com/odpf/columbus/blob/main/store/elasticsearch/record_repository.go#L208